vimiv/api/working_directory.py

```diff
@@ -224,7 +226,7 @@ def _get_content(self, directory: str) -> Tuple[List[str], List[str]]:
         """
         show_hidden = settings.library.show_hidden.value
         paths = files.listdir(directory, show_hidden=show_hidden)
-        return files.supported(paths)
+        return files.order(files.supported(paths))
```

Comment:
This is an example where we would need to unpack, i.e. `files.order(*files.supported(paths))` (see later comment in files.py).

We may want a convenience function that does
```python
files.order(*files.supported(files.listdir(directory)))
```

Reply:
I have changed the unpacking in 14621f6eb7e14de966618e993ceec31c1438c127 and adopted the calling of `files.order` adequately.

I do not fully understand the purpose of this convenience function, since this is in essence exactly what `WorkingDirectoryHandler._get_content()` does. Or do you mean we should *move* this funcionality to `utils.files`?

Reply:
You are right, looks good to me :+1: